src/Apps/Sell/Components/BottomFormNavigation.tsx

```diff
@@ -0,0 +1,79 @@
+import { Flex, Button } from "@artsy/palette"
+import { useArtworkFormContext } from "Apps/Sell/ArtworkFormContext"
+import { useFormikContext } from "formik"
+import { useRouter } from "found"
+import styled from "styled-components"
+import { themeGet } from "@styled-system/theme-get"
+
+export const BottomFormNavigation = () => {
+  const { router } = useRouter()
+  const { actions, state } = useArtworkFormContext()
+  const { isValid, isSubmitting, submitForm } = useFormikContext()
```

Comment:
This might get a bit confusing later so wanted to call out now:

Rather than `useArtworkFormContext`, can we name it something else and more specific? The reason being, we typically reserve that sort of naming for wrapping formik context hooks, which typically need to be done in order to support proper typescript types. Eg, 

```tsx
export const useArtworkFormContext = () => {
  const formikContext = useFormikContext<ArtworkFormValues>()
  return formikContext
}
```

So having the somewhat-unrelated `useArtworkFormContext` _and_ the likely needed wrapped `useFormikContext` could lead to some confusion. 

Maybe rename to `useSellContext`?

Reply:
Realizing now looking at the rest of the PR (with the individual routes) that there's multiple formik contexts for each route. So maybe this is fine (and can pass generic directly in to useFormikContext, until we need to make it more reusable). 

That said, it would be worthwhile renaming the useArtworkFormContext to something specific to avoid future confusion.

Reply:
I renamed the context to avoid conflicting naming with the Formik/form setup.

> that there's multiple formik contexts for each route.

For any particular route there should only be one Formik context active, but I think I see what you mean about the typing. This shared component is only using the generic actions from the context but let me know if there's something else you'd suggest to tighten this up.